src/main/java/io/nats/client/api/KeyValueConfiguration.java

```diff
@@ -157,8 +170,7 @@ public Builder(KeyValueConfiguration kvc) {
          * @return the builder
          */
         public Builder name(String name) {
-            this.name = validateBucketName(name, true);
-            return this;
+            return super.name(name);
```

Comment:
I did not have to override these, but it allows me to have more specific api comments.